src/Column/DataColumn.php

```diff
@@ -120,7 +130,8 @@ public function __construct(
         public readonly array $headerAttributes = [],
         public readonly mixed $bodyAttributes = [],
         public readonly bool $withSorting = true,
-        public readonly mixed $content = null,
+        string|Stringable|int|float|callable|null $content = null,
+        public bool|null $encodeContent = null,
```

Comment:
What does `null` mean here? Should it be `bool` only?

Reply:
`true` - always encode
`false` - never encode
`null` - not encode if content is instance of `NoEncodeStringableInterface` (e.g. tags from Yii HTML), otherwise encode.

Reply:
phpdoc at https://github.com/yiisoft/yii-dataview/pull/266/files#diff-ea29063f01f9f7bc5e025239af8f281993d3bd0f747e3dbca40710d981ba01eeR92 should be adjusted.

Reply:
done